js/components/graph/Edge.js

```diff
@@ -8,46 +8,33 @@ import { refLookUp } from "../common/utils"
 export default class Edge extends React.Component {
   constructor(props) {
     super(props)
-    this.state = { status: "inactive" }
```

Comment:
Now that you've deleted the only state, you don't need to override the constructor at all.